examples/charts/boxplot.py

```diff
@@ -15,9 +15,9 @@
 
 # then, we get the countries and we group the data by medal type
 countries = df.abbr.values.tolist()
-gold = df['medals.gold'].astype(float).values
-silver = df['medals.silver'].astype(float).values
-bronze = df['medals.bronze'].astype(float).values
+gold = list(df['medals.gold'].astype(float).values)
+silver = list(df['medals.silver'].astype(float).values)
+bronze = list(df['medals.bronze'].astype(float).values)
```

Comment:
what is the purpose of this? Just to demonstrate that it works with lists? We should do that with a list literal, in a different example. No one will ever want to call `list` around a data frame like this. In fact, `df['medals.gold'].astype(float).values` is already way too complicated. Does `df['medals.silver']` no work?